Shared/sdk/CVector.h

```diff
@@ -30,13 +30,9 @@ class CVector
     float fY;
     float fZ;
 
-    struct NoInit{};
```

Comment:
For what reason was removed NoInit?

Reply:
To always have an initialized vector

Reply:
Why do you need that?

Reply:
Just to make sure nothing breaks.
Not initialized vector should be zero regardless. And its only used in client code when doing sync.

Reply:
This is the standard practice for the sake of optimization. See how vectors are implemented in Eigen, glm, UE, etc.

Reply:
Well, its only 3 variables. There should be more constructions that depend on the NoInit rule for the sake of optimalization tbf

Reply:
**NoInit** can be noticable when you need to create hundreds of 3d vectors within an `std::vector`, for example. This is relevant for 3D rendering part, for example. **NoInit** should be added in other structures in future. It would be better to bring it back, yes.